.gitignore

```diff
@@ -2,3 +2,4 @@ dist
 example/public/bundle.js
 example/public/bundle.js.map
 coverage
+node_modules
```

Comment:
I pulled this out into a separate commit on master. Hopefully this goes away soon.